### ISSUE TITLE
Save Media Now gesture not working on some images

### ISSUE THREAD

**Issue by Matthewus** [marklieberman/foxygestures#251]
Hi!
Save Media Now gesture not working on images on Twitter, if the links look like the following:
https://pbs.twimg.com/media/Deh3bIAXUAAfyxP.jpg:orig
https://pbs.twimg.com/media/Deh3bIAXUAAfyxP.jpg:large
https://pbs.twimg.com/media/Deh3bIAXUAAfyxP.jpg:small
Can this be fixed?

**Comment by marklieberman** [comment]
An error is being thrown because the filename is parsed as `Deh3bIAXUAAfyxP.jpg:orig` and `:` is not a valid character in a filename. If I just strip it, then the filename will end with `.jpgorig` which isn't ideal either. I can probably do a bit of logic to handle this situation better.

**Comment by Matthewus** [comment]
Firefox saves it as .jpg if i use the "Save Image as" command, so the gesture doesn't trigger that command?

**Comment by marklieberman** [comment]
Nope, the days when you could just re-use an existing feature in Firefox ended with web extensions. That's why we have issues like #41 now. I have to use downloads.download() API which is pretty basic. See: https://github.com/marklieberman/foxygestures/blob/master/src/background/helpers.js#L90 and https://github.com/marklieberman/foxygestures/blob/master/src/background/commands.js#L978 for relevant code.
